incremental-dom/incremental-dom.d.ts

```diff
@@ -1,6 +1,6 @@
 // Type definitions for Incremetal DOM
 // Project: https://github.com/google/incremental-dom
-// Definitions by: Basarat Ali Syed <https://github.com/basarat>
+// Definitions by: Markus Lanthaler <https://github.com/lanthaler>
```

Comment:
`Definitions by: Basarat Ali Syed <https://github.com/basarat>, Markus Lanthaler <https://github.com/lanthaler>` 

Other than that all good :rose: :+1: